include/zcbor_common.h

```diff
@@ -6,11 +6,15 @@
 
 #ifndef ZCBOR_COMMON_H__
 #define ZCBOR_COMMON_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
Shouldn't these be under inclusion of other headers?

Reply:
Ah yes, that seems more correct. Fixed now.